Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM rust:1.54.0-slim-buster as tera
```

Comment:
totally a non issue but the builder image stage in multistage images is usually called `builder`.